osbs/build/build_request.py

```diff
@@ -95,6 +98,9 @@ def set_params(self, **kwargs):
 
         self.base_image = kwargs.get('base_image')
         self.low_priority_node_selector = kwargs.get('low_priority_node_selector')
+        if kwargs.get('platform'):
+            self.platform = kwargs.get('platform')
```

Comment:
Agreed. This should be:
```
self.platform_node_selector = kwargs.get('platform_node_selector')
```
(i.e. remove the `kwargs.get('platform')` check)

The `platform` keyword is only needed to work out the node selector for that platform.